core/src/test/java/org/apache/calcite/tools/PlannerTest.java

```diff
@@ -136,6 +136,20 @@ private void checkParseAndConvert(String query,
         + "    LogicalTableScan(table=[[hr, emps]])\n");
   }
 
+
+  @Test void test() throws Exception {
```

Comment:
Unit test name is confused, please briefly describe the function of the test.

Reply:
Thanks for your remind.

Reply:
Delete it.